### DISCUSSION TITLE
Is there a way to have `WARNING` flags on `mkdocs build` (or `serve`) raise an error code?

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [squidfunk/mkdocs-material#6532] [Q&A] [answered]
In my GitHub Actions suite, I run `mkdocs build` to build my documentation. I would like it to raise an error code when any `WARNING` flags are present so that the build action can fail. Is this something that can currently be done?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @Andrew-S-Rosen,
What do you mean by `WARNING` flags?
The `--strict` (`-s`) flag should stop execution on any warning that is logged in MkDocs. `mkdocs build --strict`
You can also look at https://www.mkdocs.org/user-guide/configuration/#validation to specify the logging level for different issues with url lookup and use it together with the above.

**Reply by Andrew-S-Rosen** [reply]
Thank you, @kamilkrzyskow! You answered my question. What I meant was that when you run `mkdocs build`, sometimes a `WARNING` in the logs will appear in the stdout, and I wanted to ensure that if it did it would terminate. I wasn't sure if I should be looking here or at `mkdocs`, but I now see that it's in mkdocs. Thank you!
